### ISSUE TITLE
async-std won't compile on armv5te because of AtomicU64

### ISSUE THREAD

**Issue by jaysonsantos** [async-rs/async-std#789]
Hi there, I have some software that I run on a dd-wrt router and while trying to move away from actix to use tide I got some problems when I tried to cross compile with my app with `cross build --target armv5te-unknown-linux-musleabi --release` because arm 32 bit don't have AtomicU64.
Here is the error
```
error[E0432]: unresolved import `std::sync::atomic::AtomicU64`
 --> /cargo/registry/src/github.com-1ecc6299db9ec823/async-std-1.5.0/src/task/task_id.rs:2:25
  |
2 | use std::sync::atomic::{AtomicU64, Ordering};
  |                         ^^^^^^^^^
  |                         |
  |                         no `AtomicU64` in `sync::atomic`
  |                         help: a similar name exists in the module: `AtomicU8`

error: aborting due to previous error
```
I found this closed PR but no issue https://github.com/async-rs/async-std/pull/286

**Comment by dignifiedquire** [comment]
This is fixed on master, which uses `AtomicUsize` and will be released in 1.6.0

**Comment by jaysonsantos** [comment]
Awesome, I will close this then!
